security/pfSense-pkg-zeek/files/usr/local/pkg/zeek.inc

```diff
@@ -821,41 +815,41 @@ function zeek_sync_on_changes() {
 		$synctimeout = $zeek_sync['synctimeout'] ?: '250';
 		switch ($synconchanges) {
```

Comment:
This switch statement is missing a `default` case. The default case should always be handled, even if it's a no-op, to [avoid potential logic problems](https://cwe.mitre.org/data/definitions/478.html).

Reply:
There is a default case.